src/components/app/MenuButtons/index.css

```diff
@@ -56,8 +56,8 @@
  * both of them, so one element won't be able to have both a right and a left
  * borders with this technique.
  * We're using ::after for borders because ::before is used for icons. */
-.menuButtonsButton-hasRightBorder::after,
-.menuButtonsButton-hasLeftBorder::after {
+.menuButtonsButton-hasRightBorder::before,
+.menuButtonsButton-hasLeftBorder::before {
```

Comment:
Any reason you decided to change this to `::before` in this patch? See the comment above especially. For now we don't have any "with icon" button that also has a border, so this doesn't regress anything, but this can in the future. (This may also be a good argument in favor of implementing the borders differently as I suggest below).